airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/http_requester.py

```diff
@@ -155,3 +168,343 @@ def cache_filename(self) -> str:
     def use_cache(self) -> bool:
         # FIXME: this should be declarative
```

Comment:
nit: can you update this comment? request caching isn't possible with this requester because [it doesn't use request_cache like HttpStream does](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py#L41)

Reply:
otherwise I'd also be fine removing the methods from the interface

Reply:
I will remove it for now